### PR TITLE
Update `.scale()` and `.adapt()` docstrings

### PR DESCRIPTION
**Pull request by andersy005**
Addresses https://github.com/dask/dask-jobqueue/issues/344#issuecomment-537092345

I am not sure what's the right way to include the information from `distributed's SpecCluster` without copying and pasting it into the new docstrings:

- `adapt()`: Updated
```python
In [4]: cluster.adapt?                                                                                          
Signature:
cluster.adapt(
    *args,
    minimum_jobs: int = None,
    maximum_jobs: int = None,
    **kwargs,
)
Docstring:
Scale Dask cluster automatically based on scheduler activity.

Parameters
----------

minimum_jobs : int
   Minimum number of jobs
maximum_jobs : int
   Maximum number of jobs
**kwargs :
   Extra parameters to pass to dask.distributed.Adaptive
```

- `adapt()`: Old version

```python
In [4]: cluster.adapt?                                                                                          
Signature:
cluster.adapt(
    minimum_cores=None,
    maximum_cores=None,
    minimum_memory=None,
    maximum_memory=None,
    **kwargs,
)
Docstring:
Turn on adaptivity
For keyword arguments see dask.distributed.Adaptive
Instead of minimum and maximum parameters which apply to the number of
worker, If Cluster object implements jobqueue_worker_spec attribute, one can
use the following parameters:
Parameters
----------
minimum_cores: int
    Minimum number of cores for the cluster
maximum_cores: int
    Maximum number of cores for the cluster
minimum_memory: str
    Minimum amount of memory for the cluster
maximum_memory: str
    Maximum amount of memory for the cluster
Examples
--------
>>> cluster.adapt(minimum=0, maximum=10, interval='500ms')
>>> cluster.adapt(minimum_cores=24, maximum_cores=96)
>>> cluster.adapt(minimum_memory='60 GB', maximum_memory= '1 TB')
```

- scale
```python
                                                                      
In [3]: cluster.scale?                                                                                          
Signature: cluster.scale(n=None, jobs=0, memory=None, cores=None)
Docstring:
Scale cluster to specified configurations.

Parameters
----------
n : int
   Target number of workers
jobs : int
   Target number of jobs
memory : str
   Target amount of memory
cores : int
   Target number of cores
```

### CODE REVIEW

File: dask_jobqueue/core.py [hunk 1]
```diff
@@ -498,6 +512,23 @@ def scale(self, n=None, jobs=0, memory=None, cores=None):
     def adapt(
         self, *args, minimum_jobs: int = None, maximum_jobs: int = None, **kwargs
     ):
+        """ Scale Dask cluster automatically based on scheduler activity.
+
+        Parameters
+        ----------
+
```
Comment (jhamman) [inline dask_jobqueue/core.py]: I think we should also include the minimum/maximum arguments here. I suspect most users will interact with these regularly and I think chasing the arguments through to dask.distributed.Adaptive will end up being annoying.

**Comment by andersy005** [comment]
@jhamman, 

When you get time, this is ready for another look.

File: dask_jobqueue/core.py [hunk 2]
```diff
@@ -498,6 +512,30 @@ def scale(self, n=None, jobs=0, memory=None, cores=None):
...
         self, *args, minimum_jobs: int = None, maximum_jobs: int = None, **kwargs
     ):
+        """ Scale Dask cluster automatically based on scheduler activity.
+
+        Parameters
+        ----------
+        minimum_cores : int
+           Minimum number of cores for the cluster
+        maximum_cores : int
+           Maximum number of cores for the cluster
```
Comment (jhamman) [inline dask_jobqueue/core.py]: I don't think we want the `_cores` in these names. These will be passed through to https://distributed.dask.org/en/latest/api.html#adaptive which just uses `minimum`/`maximum`.

Comment (andersy005) [reply dask_jobqueue/core.py]: 👍 Fixed it ( I think :))

**Pull request merged**